release notes/v0.32.0.md

```diff
@@ -0,0 +1,89 @@
+
+Some notable updates:
+- goja, the JS engine we use got support for let/const which let us disable a babel plugin. Previous to this if you had a particularly *long* script sometimes babel took upwards for 30 minutes to transpile. Now even our worst contender taking 51minutes took less than a minute :tada:. Also `globalThis` is now available.
+- updating the gRPC libraries fixed bug [#1928](https://github.com/k6io/k6/issues/1928) and probably others. [#1937](https://github.com/k6io/k6/pull/1937)
+
+## Other enhancements and UX improvements
+
+- JS: `ArrayBuffer` is now supported for binary data throughout the codebase, including in WebSocket messages. [#1841](https://github.com/k6io/k6/pull/1841)
+
+  It's now possible to send binary WS messages with the `socket.sendBinary()` function and to receive binary messages with the `binaryMessage` event handler:
+
+  ```javascript
+  const binFile = open('./file.pdf', 'b');
+
+  export default function () {
+    ws.connect('http://wshost/', function(socket) {
+      socket.on('open', function() {
+        socket.sendBinary(binFile);
+      });
+
+      socket.on('binaryMessage', function(msg) {
+        // msg is an ArrayBuffer, so we can wrap it in a typed array directly.
+        new Uint8Array(msg);
+      });
+    });
+  }
+  ```
+
+  For the other backwards incompatible changes see below.
+
+- Options: K6 should now warn you on unrecognised configuration js options in most cases instead of just silently ignoring them. [#1919](https://github.com/k6io/k6/pull/1919)
+- error_code: the tag `error_code` should now be set more accurately in some cases. [#1952](https://github.com/k6io/k6/pull/1951)
+
+## Bugs fixed!
+
+- Arrival rate executors could in some cases report twice as many used VUs then what was true. [#1954](https://github.com/k6io/k6/issues/1954) fixed by [#1955](https://github.com/k6io/k6/pull/1955)
+- The newly added `responseCallback` in v0.31.0, would in cases of a response erroring mid reading the body be evaluated with the returned status code, while the reported one will be `0` as the response errored out and k6 does not return half response. Now responseCallback will receive status `0`. [#1962](https://github.com/k6io/k6/pull/1962)
+- Fix kafka output not being usable with influxdb format after v0.31.0 changes. [#1914](https://github.com/k6io/k6/pull/1914)
+- Error out with a nice message if `ramping-vus` executor would've not run a single iteration instead of just doing nothing. [#1942](https://github.com/k6io/k6/pull/1942)
+
+## Internals
+
+- JS: Way for js modules to have per VU initialization and object. This can also be used by xk6 modules by implementing `github.com/loadimpact/k6/js/modules#HasModuleInstancePerVU`. Part of [#1911](https://github.com/k6io/k6/pull/1911)
```

Comment:
```suggestion
- JS: Added a way for JS modules to have per VU initialization and object. This can also be used by xk6 modules by implementing `github.com/loadimpact/k6/js/modules#HasModuleInstancePerVU`. (Part of [#1911](https://github.com/k6io/k6/pull/1911))
```